### DISCUSSION TITLE
%90 i 6 ay kilitli kalacak %10 u satilabilecek

### DISCUSSION THREAD

**Discussion by Onraygn** [IBAX-io/go-ibax#219] [Bounty Hunter]
Bu %90 6 ay sonra ne yapar zengin oluyormuyux 😂

**Comment by Serapr** [comment]
Umarım oluruz ama sizce kaçtan listelenecek 😀

**Reply by Onraygn** [reply]
🤔 bilmiyorum sadece tahmin olur ama guzel zamanda listeleniyor, eger bu 6 aylik donemde btc toparlar guzel bir boga yasanirsa harika olur. 🚀🚀🚀 Umutlar guzel insallah guzel olacak🔥🔥🔥
